docs/DeveloperGuide.md

```diff
@@ -882,13 +927,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 4a. The user has not logged in to his default email app.
+* 3a. The user has not logged in to his default email app.
```

Comment:
This should be after step 4 because `User will be taken to the sign-in page of his default email app.` happens after the email app is opened